geometric_kernels/utils/utils.py

```diff
@@ -356,3 +357,35 @@ def binary_vectors_and_subsets(d: int):
             i += 1
 
     return x, combs
+
+
+def _check_field_in_params(params, field):
+    """
+    Raise an error if `params` does not contain a `field`.
+    """
+    if field not in params:
+        raise ValueError(f"`params` must contain `{field}`.")
+
+
+def _check_1_vector(x, desc):
```

Comment:
Maybe `_check_1_el_vector` for further clarity and to avoid confusion with the next check below?